app_dart/lib/src/service/scheduler.dart

```diff
@@ -279,13 +279,6 @@ class Scheduler {
     String reason = 'Newer commit available',
     List<String>? builderTriggerList,
   }) async {
-    // Always cancel running builds so we don't ever schedule duplicates.
-    log.fine('Attempting to cancel existing presubmit targets for ${pullRequest.number}');
```

Comment:
Is this diff expected? Won't it cause duplicates to be scheduled?

Reply:
Hmm... Looking at it from the point of view of subscription I don't think it as the cancel builds call was pull out so the method is really only triggering try builds and not cancelling then triggering try builds. I suppose I can add the cancel calls to the other spots where trigger try builds is called. That way the method agreement is only triggering presubmit builds.

Reply:
Ah I will add it back. The calls come from multiple entry points into the service expecting this behavior. I can refactor it later.